src/robotide/ui/tagdialogs.py

```diff
@@ -161,8 +170,7 @@ def _search_for_tags(self):
         isreversed = (self.sort_state[1] != 1)
         self.total_test_cases = len(self._test_cases)
         self._results = sorted(self._unique_tags.items(),
-                               key=lambda x: str(x[0]).lower,
-                               reverse=isreversed)
+                               key=lambda x: str(x[0]).lower, reverse=isreversed)
```

Comment:
1. I don't think this kind of changes should be done in general.

2. I actually like the original formatting better.

3. That `str(x[0])` call looks dangerous. It will fail if any `x[0]` contains non-ASCII characters. Are all these items always Unicode or byte strings? If yes, just using `x[0].lower` would work.

4. `item` would be a better name than `x` when sorting `.items()`.